scripts/scripts.js

```diff
@@ -473,10 +475,15 @@ export function normalizeHeadings(el, allowedHeadings) {
  * Set template (page structure) and theme (page styles).
  */
 function decorateTemplateAndTheme() {
+  const addClasses = (elem, csv) => {
+    csv.split(',').forEach((v) => {
```

Comment:
Would `classes` (or `list`) make more sense than `csv` here?

Reply:
sounds great, i changed it to `classes`